Makefile

```diff
@@ -97,3 +97,19 @@ endif
 
 $(FNAME_$(TARGET)): $(SOURCES) $(XFILES)
 	$(CXX) $^ -std=c++98 $(CFLAGS_G) $(MOREFLAGS) $(OPTFLAGS) $(CPPFLAGS) $(CFLAGS) $(CXXFLAGS) $(LFLAGS) -o$@
+
+ifneq ($(findstring linux,$(targetmachine)),)
+  ifeq ($(TARGET),gtk)
+install:
+	mkdir -p $(DESTDIR)$(prefix)/bin
+	mkdir -p $(DESTDIR)$(prefix)/share/applications
+	mkdir -p $(DESTDIR)$(prefix)/share/icons/hicolor/scalable/apps
+	mkdir -p $(DESTDIR)$(prefix)/share/icons/hicolor/symbolic/apps
+	mkdir -p $(DESTDIR)$(prefix)/share/metainfo
+	install -p -m755 $(FNAME_$(TARGET)) $(DESTDIR)$(prefix)/bin
+	install -p -m755 data/com.github.Alcaro.Flips.desktop $(DESTDIR)$(prefix)/share/applications
```

Comment:
I'm not completely sure if com.github.Alcaro.Flips is the best name (I have no particular attachment to github, I may end up moving), but the alternative would be se.muncher.Flips which I'm not a too big fan of either (99% of reversed domains start with com or org, anything else feels wrong).

I guess com.github is the less bad one...

Reply:
If all apps are com.github.UserName.Project, it's not great either. :p Maybe the best is tto ask the Flathub maintainers for advices.

Reply:
I discussed about that at FOSDEM, what we need for an app to be pusblished on Flathub is a valid domain name. It's pretty common for apps with no website to use their forge's address, e.g. com.github.user.Project: https://github.com/flathub?utf8=%E2%9C%93&q=github. And nobody complained on the Flathub MR, so I think we're good here. :)